cmd/elastic.sh

```diff
@@ -3,10 +3,25 @@ set -e;
 
 function elastic_schema_drop(){ compose_run 'schema' node scripts/drop_index "$@" || true; }
 function elastic_schema_create(){ compose_run 'schema' ./bin/create_index; }
+
+# perform pre-start checks and start the elasticsearch container
 function elastic_start(){
   mkdir -p $DATA_DIR/elasticsearch
   # attemp to set proper permissions if running as root
   chown $DOCKER_USER $DATA_DIR/elasticsearch 2>/dev/null || true
+
+  # record the owner of the Elasticsearch directory
+  elasticsearch_owner_uid=$(stat -c '%u' $DATA_DIR/elasticsearch)
```

Comment:
This command doesn't seem to be available on `darwin`:

```bash
stat -c '%u' /data/pelias/docker/projects/portland-metro/elasticsearch
stat: illegal option -- c
usage: stat [-FlLnqrsx] [-f format] [-t timefmt] [file ...]
```

Reply:
```bash
uname -a
Darwin Peters-MacBook-Pro.local 18.0.0 Darwin Kernel Version 18.0.0: Wed Aug 22 20:13:40 PDT 2018; root:xnu-4903.201.2~1/RELEASE_X86_64 x86_64
```

Reply:
Good catch. I've updated that line to use the `stat --format` option, for which `-c` was a shortcut that appears to only be in the GNU coreutils version. I tested it on my Macbook so it should work for you now.